filters/shedder/admission_test.go

```diff
@@ -0,0 +1,199 @@
+package shedder
+
+import (
+	"math/rand"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+	"time"
+
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/filters"
+	"github.com/zalando/skipper/net"
+	"github.com/zalando/skipper/proxy/proxytest"
+)
+
+func TestAdmissionControl(t *testing.T) {
+	for _, ti := range []struct {
+		msg                        string
+		mode                       string
+		d                          time.Duration
+		windowsize                 int
+		minRequests                int
+		successThreshold           float64
+		maxrejectprobability       float64
+		aggression                 float64
+		N                          int     // iterations
```

Comment:
I think it is not used

Reply:
true, but maybe I should change the outer for loop to N<ti.N

I think I will do it.